### PR TITLE
Update config-settings.rst

### PR DESCRIPTION
**Pull request by james-mm**
Add string configuration to enable SSL for database connections

### CODE REVIEW

**Comment by jasonblais** [comment]
Made a formatting tweak for the note, and applied it throughout the document.

Also tweaked the wording a little. @lindy65 can you help review the change for correctness?

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2618,7 +2626,10 @@ This setting can only be changed from config.json file, it cannot be changed fro
 
 Data Source
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-This is the connection string to the master database. When **DriverName** ="postgres" then use a connection string in the form ``postgres://mmuser:password@localhost:5432/mattermost_test?sslmode=disable&connect_timeout=10``. This setting can only be changed from config.json file, it cannot be changed from the System Console user interface.
+This is the connection string to the master database. When **DriverName** is set to ``postgres``, use a connection string in the form ``postgres://mmuser:password@localhost:5432/mattermost_test?sslmode=disable&connect_timeout=10``. This setting can only be changed from ``config.json`` file.
+
+.. note::
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: Devs: You only need to review this line

Comment (lindy65) [reply source/administration/config-settings.rst]: Looks good to me Jason!

Comment (jespino) [reply source/administration/config-settings.rst]: Maybe "add ``&tls=true``" will be more explicit about what exactly you need to add to the string. But anyway, is ok for me. :)

Comment (jasonblais) [reply source/administration/config-settings.rst]: @jespino Good point, updated

**Pull request merged**